src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -43,7 +43,25 @@
  * <pre>
  * &lt;module name=&quot;MissingSwitchDefault&quot;/&gt;
  * </pre>
- *
+ * <p>Example of violation: </p>
+ * <pre>
+ *switch (i) {
```

Comment:
this line is missing a space indentation like the line above it. Javadocs should always have **atleast** 1 space after the leading asterisk when the line has text.